plugins/mcreator-localization/help/default/biome/base_height.md

```diff
@@ -1,7 +1,7 @@
-This parameter controls the height at which the biome is. 
-Mountains have this parameter set to a big value for example.
-
-This parameter also controls placement of the biomes.

```

Comment:
Keep plural here. It controls placement of all biomes, if you change settings of one biome, all biomes are indirectly affected

Reply:
Missed that, thanks for heads-up.